analyzers/src/SonarAnalyzer.CFG/ShimLayer/OperationInterfaces.xml

```diff
@@ -2,7 +2,7 @@
 <!-- Licensed to the .NET Foundation under one or more agreements. The .NET Foundation licenses this file to you under the MIT license. See the LICENSE file in the project root for more information. -->
 <Tree Root="IOperation">
```

Comment:
did you suggest this change upstream in StyleCopAnalyzers? 

https://github.com/DotNetAnalyzers/StyleCopAnalyzers/blob/05489698155b609dfc081c06ad21771ddcf26f6c/StyleCop.Analyzers/StyleCop.Analyzers/Lightup/OperationInterfaces.xml


Let's not diverge our own copies without at least suggesting the change to them

Reply:
A year ago, our generator had problems with the updated version of Operations.xml (because of nullable annotations) The upstream repository hadn't been updated at that point. Since then a lot has changed in the upstream repo and we are probably better off using the upstream source generator.